csv_to_vanilla-i18n/csv_to_json.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path= ""  // enter your data file path here
```

Comment:
Can we please ask for user input here?

Ref: https://www.codecademy.com/articles/getting-user-input-in-node-js